.gitignore

```diff
@@ -1,11 +1,11 @@
-/tf
-/test
-/example
-/.dapper
-/.cache
-/bin
-/build
-/dist
-*.swp
-.idea
+/tf
+/test
+/example
+/.dapper
+/.cache
+/bin
+/build
+/dist
+*.swp
+.idea
 .envrc
```

Comment:
Nit: is the addition of `.envrc` deliberate?

Reply:
it is not an addition ;) it is the only line that doesn't have the character